examples/browser-api-playground/server.js

```diff
@@ -36,10 +34,6 @@ app.post("/generateAnonymousAuthToken", (req, res) => {
   axios({
     method: "post",
     url: `${backend}/api/v1/createanonymoustoken`,
-    headers: {
```

Comment:
Intentional? Seems reasonable just curious how it worked earlier or does the `createanonymoustoken` ignores the removed headers

Reply:
`createanonymoustoken` doesn't require both actAs and apiKey. I don't know why I added it in first place. Probably, I just copied from `generateAuthToken`.